bridge_adaptivity/config/settings/base.py

```diff
@@ -143,3 +145,16 @@
 PROBLEM_ACTIVITY_TYPES = (
     'problem',
 )
+
+# Django Rest Framework
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.TokenAuthentication',
+        # for browsable api view usage
+        'rest_framework.authentication.SessionAuthentication',
+    ),
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
+    'TEST_REQUEST_DEFAULT_FORMAT': 'json'
```

Comment:
why not to add comma in the end of this line?

Reply:
👍 nice catch